### ISSUE TITLE
You have requested a non-existent service "huh.request"

### ISSUE THREAD

**Issue by LBeckX** [heimrichhannot/contao-multifileupload-bundle#59]
**Context**
Contao version: 4.13.29
Bundle version:
PHP version: 8.1

**Description**
After installation and create a multifile upload field a get the following error:

```
[2023-09-11T15:21:58.783895+02:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException:
"You have requested a non-existent service
"huh.request"."
at /app/vendor/symfony/dependency-injection/Container.php line 272 {"exception":"[object] 

(Symfony\\Component\\DependencyInjection\\Exception\\ServiceNotFoundException(code: 0): You have requested a non-existent service \"huh.request\". at /app/vendor/symfony/dependency-injection/Container.php:272)"} []
```

**Comment by koertho** [comment]
Looks like a missing dependency on heimrichhannot/contao-request-bundle

**Comment by LBeckX** [comment]
Ticket created

**Comment by Misco** [comment]
what is the solution? Thanks

**Comment by Misco** [comment]
ok installing contao request bundle =)

**Comment by koertho** [comment]
Should be fixed in version 1.8.6. You can uninstall request bundle :)
